SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -30,6 +32,7 @@ void FSpatialGDKEditorModule::StartupModule()
 	RegisterSettings();
 
 	ExtensionManager->RegisterExtension<FGridLBStrategyEditorExtension>();
+	SpatialGDKEditorInstance = MakeShareable(new FSpatialGDKEditor());
```

Comment:
Really not sure about this, don't think we should be making instances of the editor so we can get... wait what even is `GetPackageAssemblyRef` and why is that a shared pointer in the Editor

Reply:
GetPackageAssemblyRef is used to get the pointer of `FSpatialGDKPackageAssembly`,  so I can get the assembly build status. The reason I used SharedPtr here is because the old SpatialGDKEditorInstance point in `FSpatialGDKEditorToolbarModule` is a SharedPtr, there are many places are using this, in order to reducing the impact to current code, I use SharedPtr at new position as well.